.lintr

```diff
@@ -1,7 +1,7 @@
 linters: linters_with_defaults(
   line_length_linter = line_length_linter(120L),
   object_name_linter = object_name_linter(styles = c("snake_case", "symbols", "CamelCase")),
-  cyclocomp_linter = cyclocomp_linter(30L),
+  cyclocomp_linter = NULL, # Issues with R6 classes.
```

Comment:
We can also get rid of `any::cyclocomp` here too, yeah? https://github.com/posit-dev/connectapi/blob/e8c807523977158f1817abea1cce02da5097a63e/.github/workflows/lint.yaml#L25-L26

Reply:
Done.